src/test/java/gameframework/motion/overlapping/mocks/OverlapProcessorDefaultImplMock.java

```diff
@@ -0,0 +1,11 @@
+package gameframework.motion.overlapping.mocks;
+
+import gameframework.game.GameEntity;
+import gameframework.motion.overlapping.OverlapProcessorDefaultImpl;
+
+public class OverlapProcessorDefaultImplMock extends OverlapProcessorDefaultImpl {
+
+	public Boolean test_isAdding(GameEntity entity) {
```

Comment:
the method is misnamed. It does not test anything about addition. It tests if something is contained by something else. Please rename.